UI: CE Datepicker (#30488)
hashicorp/vault: changelog/30488.txt

```release-note:improvement

> ui (community edition): Update date selector in client count usage page to disable current month selection.

Reviewer: I have not been privy to many of the conversations about this feature, but I know the changelog pattern is to warn of enterprise-only changes, but not community-only changes. I believe that's because there's the general assumption that what is on community is also on enterprise, and enterprise features are only additive. 

All that to say, do we need to call out (community edition) here? Maybe this is a product-specific question.

Reviewer 2: Pretty sure we just do this for "enterprise", but might be worth updating the copy accordingly
```suggestion
ui: Update date selector in client count usage page to disable current month selection for Vault cluster's without a license.
```
